src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
+
+==== Native Scheduler Node Source Configuration
+
+Using the <<_glossary_rm_web_interface,*Resource Manager Web Interface*>>, you can create a <<_glossary_node_source,*Node Source*>> used to aquire <<_glossary_proactive_node,*ProActive Nodes*>> from a Native Scheduler.
+
+From The drop down menu, the *NativeSchedulerInfrastructure* and *NativeSchedulerPolicy* must be selected.
+
+Here is an explanation of the node source parameters:
+
+ * `Name` : you should name the node source accordingly to your infrastructure, it can be the cluster name, head node name, or native scheduler name (PBS, SLURM), etc.
+
+===== NativeSchedulerInfrastructure
+
+ * `RMCredentialsPath` : point to a file which contains the credentials of an administrator user which will own the node source. The ProActive Scheduler Server release contains two admin users credentials files : `config/authentication/rm.cred` and `config/authentication/admin_user.cred`
+ * `NSFrontalHostAddress` : the host name or IP address of the cluster head node.
+ * `NSSchedulerHome` : the location of the shared ProActive installation on cluster nodes (cluster nodes must be able to access ProActive libraries in order to start ProActive Node). Example `/opt/proactive/activeeon_enterprise-node-linux-x64-8.1.0`.
+ * `javaHome` : similarly, cluster nodes must be able to access the java command in order to start ProActive Nodes. ProActive installation includes a Java Runtime Environment under the `jre` subfolder. Example: `/opt/proactive/activeeon_enterprise-node-linux-x64-8.1.0/jre`.
+ * `jvmParameters` : additional options which can be passed to the java command.
+ * `sshOptions` : additional options which can be passed to the ssh command used to connect to NSFrontalHostAddress.
+ * `NSNodeTimeoutInSeconds` : timeout to wait for the deployment of ProActive Nodes on the cluster. As it the time needed to deploy ProActive Nodes depends on the cluster load, this timeout should be a large value.
+ * `ìmpersonationMethod`: when a job is submitted to the native scheduler, the submission is performed under the current ProActive Scheduler user. An impersonation is thus performed between the scheduler process user and the target user.
+ This impersonation can be performed using 3 different strategies:
+ ** `ssh`: in that case the head node is contacted using a ssh command with the current ProActive Scheduler user and password. User/password combination between the ProActive Scheduler and the head node operating system must match.
+ ** `none`: in that case the head node is contacted using a ssh command with the ProActive Scheduler process user (passwordless ssh). Submission to the native scheduler will be performed with the same account.
+ ** `sudo`: similar to `none` regarding the connection to the head node, but a `sudo` command will be initiated to impersonate as the current Scheduler User, before doing a job submission.
```

Comment:
Do you really mean the `Scheduler User` here? Or do you mean the `ProActive Scheduler User` (completely different meaning)?

Reply:
same remark